app/src/test/java/org/simple/clinic/appconfig/AppConfigRepositoryTest.kt

```diff
@@ -30,4 +36,83 @@ class AppConfigRepositoryTest {
         .assertValue(FetchSucceeded(countries))
         .assertNoErrors()
   }
+
+  @Test
+  fun `failed network calls to fetch the app manifest to return an appropriate failure result`() {
```

Comment:
Rename to `failed network calls to fetch the app manifest should return network related failure result`